CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 ## <sub>main</sub>
 #### _Unreleased_
 
-## <sub>v3.0</sub>
+## <sub>v3.0.1</sub>
 #### _Unreleased_
 
+**Bugfixes**
+* Fixed load error in Selenium Webdriver Edge
+  * This issue was due to Selenium changing their path for Edge Options during the V4 betas
+
```

Comment:
spare newline here

Reply:
I removed the extra line, but I did not get what do you mean with:

> lets bump the version here and now to save time.